libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1390,17 +1396,37 @@ void ExpressionCompiler::appendBitOperatorCode(Token::Value _operator)
 	}
 }
 
-void ExpressionCompiler::appendShiftOperatorCode(Token::Value _operator)
+void ExpressionCompiler::appendShiftOperatorCode(Token::Value _operator, Type const& _leftType, Type const& _rightType)
 {
-	BOOST_THROW_EXCEPTION(UnimplementedFeatureError() << errinfo_comment("Shift operators not yet implemented."));
+	// stack: rvalue lvalue
```

Comment:
Please don't use the words `rvalue` and `lvalue` because the `lvalue` might not be an lvalue.